plugin/src/main/java/com/denizenscript/denizen/scripts/commands/entity/InvisibleCommand.java

```diff
@@ -73,87 +96,130 @@ else if (!scriptEntry.hasObject("target")
                     && Utilities.entryHasNPC(scriptEntry)) {
                 scriptEntry.addObject("target", Utilities.getEntryNPC(scriptEntry).getDenizenEntity());
             }
-            else if (!scriptEntry.hasObject("target")
-                    && arg.matchesArgumentType(EntityTag.class)) {
-                scriptEntry.addObject("target", arg.asType(EntityTag.class));
-            }

```

Comment:
This reordering likely willl break the legacy `player` input support

Reply:
Reverted